tests/host_nodes_test.py

```diff
@@ -0,0 +1,10 @@
+
+from htcanalyze.host_nodes import node_cache
+
+
+def test_node_cache():
+    node_cache.gethostbyaddrcached("172.217.0.0")
+    assert node_cache.rdns_cache["172.217.0.0"] == "ord38s04-in-f0.1e100.net"
```

Comment:
This might change based on provider, right? Probably not good persistent value for testing.

Reply:
Yes that's a total good point which I also had in mind. I just don't know if there are providers for something like this, so I am just assuming, that google will keep this one a while. But as you questioned it, I will add a output and description, so whenever this fails, the test can be fixed.
Else if you know like an address which is "kind of" made for those test purposes, please let me know.